src/schema/filter_artworks.js

```diff
@@ -99,17 +100,26 @@ export const FilterArtworksType = new GraphQLObjectType({
         { rootValue: { filterArtworksLoader } }
       ) => {
         const relayOptions = parseRelayOptions(args)
+
         return filterArtworksLoader(
           assign(gravityOptions, relayOptions, {})
         ).then(({ aggregations, hits }) => {
           if (!aggregations || !aggregations.total) {
             throw new Error("This query must contain the total aggregation")
           }
 
-          return connectionFromArraySlice(hits, args, {
-            arrayLength: aggregations.total.value,
-            sliceStart: relayOptions.offset,
-          })
+          return assign(
+            {
+              pageCursors: createPageCursors(
+                relayOptions,
+                aggregations.total.value
+              ),
+            },
+            connectionFromArraySlice(hits, args, {
+              arrayLength: aggregations.total.value,
+              sliceStart: relayOptions.offset,
+            })
+          )
```

Comment:
Not sure if this works in our current version of Node, but in Chrome, you can flatten objects without a library:

```javascript
return {
  pageCursors: createPageCursors(
    relayOptions,
    aggregations.total.value
  ),
  ...connectionFromArraySlice(hits, args, {
    arrayLength: aggregations.total.value,
    sliceStart: relayOptions.offset,
  })
}
```

Reply:
@acjay This code is transpiled, so your suggestion works 👍